docker/utils.sh

```diff
@@ -48,8 +48,8 @@ dvc_pull_models() {
   fi
 
   # Pull the NER models in DVC
-  pushd "$DATA_AND_MODELS/pipelines/ner/" || exit
-  grep -Eo '\badd_er_[0-9]+\b' dvc.yaml | xargs dvc pull
+  pushd "$DATA_AND_MODELS/models/ner_er/" || exit
+  grep -E '^/model-' .gitignore | tr -d '/' | xargs dvc pull
```

Comment:
Ah nice! Good finding! It reduces the number of models downloaded by 2, right ?

Reply:
Well actually the old grep doesn't work anymore because we're now using `foreach` and the actual model names are listed in `params.yaml`. I could've also grepped `params.yaml`, but going for `.gitignore` seemed a little bit more straighforward.